### DISCUSSION TITLE
Cannot install solidus_i18n

### DISCUSSION THREAD

**Discussion by lamoureuxmarie** [solidusio/solidus#4714] [Troubleshooting] [answered]
Hi all, 

I am trying to install the solidus_i18n gem to translate my store. 
Unfortunately after I add the following gems

```
gem 'solidus_i18n', '~> 2.0'
gem 'rails-i18n', '~> 5.1'
gem 'kaminari-i18n', '~> 0.5.0'
```
I get this error message

```
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    rails (~> 7.0.4) was resolved to 7.0.4, which depends on
      railties (= 7.0.4)
```

Can anyone help please ? :)

**Comment by jarednorman** [comment, marked as answer]
`rails-i18n` version 5.1 doesn't support Rails 7. You need to use a newer version.

**Comment by lamoureuxmarie** [comment]
Oh, of course 😅 Thanks Jared!
